__tests__/components/ResourceCreation/PatientCreation.test.tsx

```diff
@@ -4,11 +4,32 @@ import { mantineRecoilWrap, getMockRecoilState } from '../../helpers/testHelpers
 import PatientCreation from '../../../components/ResourceCreation/PatientCreation';
 import { patientTestCaseState } from '../../../state/atoms/patientTestCase';
 import { download } from '../../../util/downloadUtil';
+import { selectedPatientState } from '../../../state/atoms/selectedPatient';
+import { measureBundleState } from '../../../state/atoms/measureBundle';
+import testBundle from '../../fixtures/bundles/MockMeasureBundle.json';
+import MeasureUpload from '../../../components/MeasureUpload';
+import { Calculator } from 'fqm-execution';
+import { act } from 'react-dom/test-utils';
```

Comment:
I _think_ it's the same thing (not sure), but we usually import `act` from `@testing-library/react` for consistency. Can you just add act to the existing import on line 1 and remove this statement? It should work the same